pkg/systemd/overview.less

```diff
@@ -1,4 +1,3 @@
-@import "../../node_modules/@patternfly/react-styles/css/components/Table/table.css";
```

Comment:
I see you moved this to be an import in the .jsx file. Is that generally what we should be doing, or should we conceptually have an overview.scss file and import PF/table.scss there? Is that more efficient or more debuggable in any way?

Reply:
Nevermind, a subsequent commit moves it back.